### ISSUE TITLE
Use process.hrtime instead of Date.now()

### ISSUE THREAD

**Issue by popomore** [jstrace/jstrace#12]
http://nodejs.org/api/process.html#process_process_hrtime

**Comment by tj** [comment]
yeah good call. Although even transmitting the json alone takes quite a bit of time, this won't be great for high resolution monitoring right now, maybe once #10 is implemented

**Comment by tj** [comment]
k added that, we could add this in addition to the timestamp
